packages/storybook/src/nimble/tree-view/tree-view.stories.ts

```diff
@@ -33,12 +35,14 @@ interface AnchorItemArgs {
 }
 
 const selectionModeDescription = `
-<li>All: all items in the tree are selectable through user interaction</li>
-<li>Leaves only: only the leaf items in the tree are selectable through user interaction</li>
-<li>None: no items in the tree are selectable through user interaction</li>
+<li>all: all items in the tree are selectable through user interaction</li>
+<li>leaves-only: only the leaf items in the tree are selectable through user interaction</li>
+<li>none: no items in the tree are selectable through user interaction</li>
```

Comment:
For better consistency with other bulleted lists in our docs
```suggestion
<li>all: All items in the tree are selectable through user interaction</li>
<li>leaves-only: Only the leaf items in the tree are selectable through user interaction</li>
<li>none: No items in the tree are selectable through user interaction</li>
```